src/tools/translate-to-fuzz.h

```diff
@@ -614,8 +614,8 @@ class TranslateToFuzzReader {
           }
         }
         switch (conditions) {
-          case 0: if (!oneIn(4)) continue;
-          case 1: if (!oneIn(2)) continue;
+          case 0: if (!oneIn(4)) continue; [[clang::fallthrough]];
```

Comment:
this is a clang-specific extension, I guess? it would mean we only build on clang though, how about adding a `break;` on these lines, would they fix the warning?